mapbox_vector_tile/encoder.py

```diff
@@ -251,8 +272,8 @@ def _handle_skipped_last(self, f, skipped_index, cur_x, cur_y, x_, y_):
 
     def _parseGeometry(self, shape):
         coordinates = []
-        line = "line"
-        polygon = "polygon"
+        lineType = "line"
+        polygonType = "polygon"
```

Comment:
Are the changes in this function changing any behavior? Seems like just the variables have been renamed, which I'm not opposed to, just wondering if I'm missing something.